### DISCUSSION TITLE
Working with JS Front-End

### DISCUSSION THREAD

**Discussion by undjike** [lucidarch/lucid#15] [Ideas]
Hello team!

What if we think of implementing the logic for **JS front-end** like **VueJS**? Cause when working with them, normally, the initilization comes in the `ressources/js/app.js` file and the views in `ressources/js/Pages` (when working with [InertiaJS](https://inertiajs.com) for instance).

Also there is a simalar logic for some **CSS Framework** like [Tailwind CSS](https://tailwindcss.com) which works with `Webpack` and `Mix` to compile assets.

**Comment by undjike** [comment]
I think Lucid can found a way to manage this type of things in such a way that it keeps easy to work with.

**Reply by Mulkave** [reply]
I agree there's a lack of front-end support in Lucid, mainly due to it being the underlying architecture, but would love to provide more elements to help in that direction.

Can you please elaborate on what you expect of Lucid to do in order to make it simpler to deal with these?

**Comment by undjike** [comment]
Of course. I will submit something as soon as possible.

**Comment by undjike** [comment]
Did I say "Happy new year", @Mulkave !?

**Reply by Mulkave** [reply]
Haha, now you did! Thank you 😄 
Happy new year to you too @undjike! Wish you a successful one 🎉

**Reply by undjike** [reply]
Thanks a lot. Wish you the same! 💫
